src/main/kotlin/no/nav/familie/ks/sak/kjerne/avstemming/domene/GrensesnittavstemmingTaskDto.kt

```diff
@@ -1,8 +1,10 @@
 package no.nav.familie.ks.sak.kjerne.avstemming.domene
 
 import java.time.LocalDateTime
+import java.util.UUID
 
 data class GrensesnittavstemmingTaskDto(
     val fom: LocalDateTime,
     val tom: LocalDateTime,
+    val avstemmingId: UUID?,
```

Comment:
Denne må være optional for bakoverkompabilitet. GrensenittavstemminScheduler leser opp den siste tasken i status Ferdig, og den har ikke avstemmingId